client/src/paths/Activities.tsx

```diff
@@ -789,7 +753,13 @@ export function Activities() {
   );
 
   return (
-    <>
+    <Box
+      data-test="Activities"
+      width="100%"
```

Comment:
It appears, that when the width is 100% and the left/right margins are 20 px, the overall width is 40 px wider than the parent container, at least when the parent container has a specified width. If you set this width to 
`width={{ base: "100%", md: "calc(100% - 40px)" }}`
then it won't spill over. (Same thing for Trash.)